.travis.yml

```diff
@@ -19,8 +19,8 @@ script: bash ci/tox/travis-script.sh
 jobs:
   include:
     - stage: Tests
-      env: PYTHON_VERSION=2.7 GIT_VERSION=1.7.10  # Earliest version of git supported by git-machete
-    - env: PYTHON_VERSION=2.7 GIT_VERSION=2.7.6
+      env: PYTHON_VERSION=3.6 GIT_VERSION=1.7.10  # Earliest version of git supported by git-machete
+    - env: PYTHON_VERSION=3.6 GIT_VERSION=2.7.6
     - env: PYTHON_VERSION=3.6 GIT_VERSION=2.20.2
```

Comment:
Probably this one (`3.6`+`2.20.2`) is now redundant... TBH I've only had two cases so far:
* either the earliest version fails on some obscure feature missing (like unknown %-format specifier in `git log --format`)
OR
* well, everything fails